frontend/deploy/helm/frontend/templates/acrpullbinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: acrpull.microsoft.com/v1beta2
+kind: AcrPullBinding
+metadata:
+  name: pull-binding
+spec:
+  acr:
+    environment: PublicCloud
+    scope: {{ .Values.pullBinding.registry }}
+    server: {{ .Values.pullBinding.scope }}
+  auth:
+    workloadIdentity:
+      serviceAccountRef: frontend
```

Comment:
The SA whose credentials are projected is not necessarily the same SA as where they are projected, on purpose. So if we wanted a different identity with pull rights federated to some other SA in the namespace, we could.

Reply:
it's easy to use a separate one now. lets do that.